spec/system/searchgov_domain_management.rb

```diff
@@ -0,0 +1,28 @@
+describe 'Searchgov Domain Management' do
+    let(:searchgov_domain) { searchgov_domains(:basic_domain) }
+  let(:params) do
+    { id: searchgov_domain.id }
+  end
+    let(:current_user) { users(:affiliate_admin) }
+    before do
+      Rails.application.env_config["omniauth.auth"] = mock_user_auth('affiliate_admin@fixtures.org')
+    end
+
+    describe 'reindexing a domain' do
+      #subject(:reindex) { post :reindex, params: params }
+
+      it 'triggers a reindex of the domain' do
+        puts "visiting login"
```

Comment:
Two things:
1) I _really_ don't like puts inside of specs. The extra noise during test runs is a problem.
2) Can we get rid of the steps that have nothing to do with the functionality under test? Login etc. aren't being tested here, and don't really belong in the test.
3) The test isn't actually testing anything; the expectation is commented out!